.changeset/eleven-spies-thank.md

```diff
@@ -0,0 +1,6 @@
+---
+'@spotlightjs/overlay': patch
+'@spotlightjs/spotlight': patch
```

Comment:
This is actually a big improvement and adds new functionality so marking it as a minor 
```suggestion
'@spotlightjs/overlay': minor
'@spotlightjs/spotlight': minor
```